config/development.json

```diff
@@ -183,6 +183,7 @@
 		"upsell/concierge-session": true,
 		"woop": false,
 		"wordpress-action-search": false,
-		"wpcom-user-bootstrap": false
+		"wpcom-user-bootstrap": false,
+		"newsletter/stats": false
```

Comment:
Usually, the flags are using a more generic label in the first part. In this case, the opposite `stats/newsletter` would fit better.

Reply:
The flags are also sorted alphabetically. Same for the other envs.